data/sql/updates/pending_db_world/hawkfix.sql

```diff
@@ -0,0 +1,10 @@
+DELETE FROM `creature_template_movement` 
```

Comment:
From what I have seen in other code reviews, these statements could all be structured to be in one line.

Reply:
Sure, I can change it if that's the preferred format, though the number of executions stays the same so for the db import the load won't be affected

Reply:
Yeah, its more about code style.